services/dbt/models/datenspende/user_research/schema.yml

```diff
@@ -2,24 +2,70 @@
 version: 2
 
 models:
-  - name: vitaldata_don_per_day
-    description: Number of daily vitaldata donations/donors per vital type
+  - name: vitaldata_per_day
+    description: Number of daily vitaldata donors of vital type per date
+    meta:
+      owner: "@davhin"
     columns:
-      - name: cdate
-        description: calendar date
+      - name: date
+        description: calendar date of vital data points
       - name: vital_type
         description: vital data type
-      - name: donors
-        description: The number of daily vitaldata donors/donations (synonymous) per vital type
+      - name: users
```

Comment:
Better call it something like `number_of_users` otherwise one might expect a list of user IDs just looking at the column name

Reply:
good point, thx